Source/WebKit/WebProcess/haiku/WebProcessHaiku.cpp

```diff
@@ -43,30 +43,37 @@ void WebProcess::platformInitializeWebProcess(WebProcessCreationParameters& para
 
 void WebProcess::platformInitializeProcess(const AuxiliaryProcessInitializationParameters&)
 {
+    notImplemented();
```

Comment:
I think there was nothing here because there was nothing to do. In that case there is no need for a notImplemented(). Likewise for platformTerminate: the method is there if you need it, but having it empty is a perfectly acceptable implementation.